src/daemon/osd_scenarios/osd_volume_activate.sh

```diff
@@ -6,7 +6,10 @@ function osd_volume_activate {
 
   # Verify the device is a valid ceph-volume device
   # If not the following command will return 1 with "No valid Ceph devices found"
-  ceph-volume lvm list "${OSD_DEVICE}"
+  if ! ceph-volume lvm list "${OSD_DEVICE}"; then
+    cat /var/log/ceph
```

Comment:
`/var/log/ceph` is a directory?

Reply:
No this is where `ceph-volume` logs end up.